spec/dry_run_spec.rb

```diff
@@ -1,12 +1,11 @@
 require 'spec_helper'
 
 describe Slackistrano do
-  %w[updating reverting updated reverted failed].each do |stage|
+  %w(updating reverting updated reverted failed).each do |stage|
```

Comment:
Heh. I can't stand this style of arrays unfortunately.  I probably should add rubocop to the project in general, but going to leave this one out for now.  Grabbed the other one though. Thx.